src/StatusQ/Controls/StatusInput.qml

```diff
@@ -359,6 +364,9 @@ Item {
             Layout.fillWidth: true
             Layout.fillHeight: true
 
+            leftPadding: root.leftPadding
+            rightPadding: root.rightPadding
```

Comment:
This will cause the padding to double no? Because AFAIK we use it already inside `StatusInput` to control padding of itself. Adding this here, would add the padding as well to StatusBaseInput which means it's doubled (unless I'm missing something) That desired?

Reply:
`StatusBaseInput` effectively has the `leftPadding` and `rightPadding` properties exposed, so if you use `StatusBaseInput` you can customize the padding properly. On the other hand, `StatusInput` already had the same properties exposed but not linked to anything, so the padding in this case was not customizable. What I've done here is simply bind these properties to the `StatusBaseInput` properties, so now `StatusInput` allows you to customize them as well.